source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -855,6 +882,9 @@ module Error = struct
         "code", `Int (code kind);
         "location", location;
       ]
+
+
+  let hash = Hashtbl.hash
```

Comment:
We should use `@@deriving hash` on `type t` instead. I'm not sure what this does?